package-lock.json

```diff
@@ -19,7 +19,7 @@
       "integrity": "sha512-rlXwdnbdLkCshUQPcE7+hqFoCX6SgaH+0ZPcOPE9f1TxP3WkiLnT2F/8D8JXV/zuVYub2IzcnzimqW7C9wEpYw==",
       "dev": true,
       "requires": {
-        "@mapbox/cloudfriend": "1.8.2"
+        "@mapbox/cloudfriend": "^1.8.1"
```

Comment:
are these package-lock changes needed or could we drop them?

Reply:
@vsmart we could drop them, but I think they're just the typical formatting changes from moving to npm >= 5.7. The behavior shouldn't be any different.